staff/clara-rufi/skylabcoders-html-semantics/community.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h1>
+            <a href="/en"><img src="/images/403/default.png" alt="Skylab Coders Academy">Skylab Coders Academy</a>
+        </h1>
+        <ul>
+            <li>
+                <a lang="es" href="/es">es</a>
+            </li>
+            <li>
+                <strong>
+                    <a lang="en" href="/en">en</a>
+                </strong>
+            </li>
+            <li>
+                <a href="/ca">ca</a>
+            </li>
+        </ul>
+        <form action="/en/search-results_21631">
+            <input placeholder="" type="search" name="q">
```

Comment:
If you don't want placeholder, you can delete it from the line, If not, it would be useful to put inside any word that clarifies what the user is supposed to write in that input

Reply:
i put an input inside placeholder, thanks :)